src/app/_components/BlogPostHeader.tsx

```diff
@@ -0,0 +1,47 @@
+import Image from 'next/image'
+
+import { Badge } from '@/components/Badge'
+import { Heading } from '@/components/Heading'
+
+import { type ImageProps } from '@/types/sharedProps/imageType'
+
+import { formatDate } from '@/utils/formatDate'
+
+type BlogPostHeaderProps = {
+  title: string
+  date?: string
+  image?: ImageProps
+  category?: string
+}
+
+export function BlogPostHeader({
+  title,
+  date,
+  image,
+  category,
+}: BlogPostHeaderProps) {
+  return (
+    <header>
+      <div className="mb-8 space-y-4">
```

Comment:
This is a layout style that belongs to the parent.

```suggestion
      <div className="space-y-4">
```